torch/csrc/jit/script/module.h

```diff
@@ -222,6 +222,8 @@ struct TORCH_API Module : public Object {
       const std::string& filename,
       const ExtraFilesMap& extra_files = ExtraFilesMap()) const;
 
+  std::vector<std::string> opnames() const;
```

Comment:
It doesn't have to be a method of `torch::jit::script::Module` class. I think it can and should live separately, and you actually already implemented it this way.